cpp/src/slice2cs/Gen.cpp

```diff
@@ -1080,6 +1080,7 @@ Slice::Gen::Gen(const string& base, const vector<string>& includePaths, const st
     _out << nl << "#pragma warning disable SA1309 // Field names must not begin with underscore";
     _out << nl << "#pragma warning disable SA1312 // Variable names must begin with lower case letter";
     _out << nl << "#pragma warning disable SA1313 // Parameter names must begin with lower case letter";
+    _out << nl << "#pragma warning disable CA1707 // Remove the underscores from member name";
```

Comment:
Why is this necessary?

Reply:
because `iceP_` param prefix

Reply:
Basically for
```
public static ZeroC.Ice.OutgoingRequestFrame Sleep(ZeroC.Ice.IObjectPrx proxy, int iceP_seconds, global::System.Collections.Generic.IReadOnlyDictionary<string, string>? context) =>
                ZeroC.Ice.OutgoingRequestFrame.WithArgs(
                    proxy,
                    "sleep",
                    idempotent: false,
                    compress: false,
                    format: default,
                    context,
                    iceP_seconds,
                    ZeroC.Ice.OutputStream.IceWriterFromInt);
```

I think should be fine to always use `args` instead of the `iceP_<>` as this is now only for the case we have a single arg.